### DISCUSSION TITLE
Export the animation of FDS output

### DISCUSSION THREAD

**Discussion by lihy213** [firemodels/fds#12591] [Q&A]
Dear contributors,

Can FDS software export the animation of its output in the type of _gif_ or _avi_? I know it can render the animation into many pictures, but it seems inconvenient. Do you know of any effective ways to be supported?

Kind regards,
LI

**Comment by TristanHehnen** [comment]
I'm not sure if SmokeView can output as a video directly, but you can easily make a video from an image sequence with many tools.

A very common one would be [ffmpeg](https://ffmpeg.org/).

I prefer to use Blender. It could be considered as a graphical user interface for ffmepg, since it is used in the background:
https://www.youtube.com/watch?v=JsiP0sY9ADQ

**Reply by gforney** [reply]
Smokeview also has a simple graphical interface for ffmpeg if ffmpeg is in the path. Look under the Dialog>file menu (after putting ffmpeg in your path)

**Reply by TristanHehnen** [reply]
I didn't realise that this was available. 
Good to know, thanks!

**Reply by lihy213** [reply]
GREAT! It works! :smiley: The principle of this method seems to be similar to what @TristanHehnen said.

**Comment by gforney** [comment]
also, if you put ffplay in the path you can play the moview you made from the same dialog

**Reply by lihy213** [reply]
Thanks, I will try it!
